lab_1_keywords_tfidf/main.py

```diff
@@ -99,12 +161,32 @@ def calculate_tfidf(term_freq: dict[str, float], idf: dict[str, float]) -> Optio
 
     In case of corrupt input arguments, None is returned
     """
-    pass
-
+    # я понятия не имею работает ли эта функция
+    if isinstance(term_freq, dict) is False or isinstance(idf, dict) is False:
+        return None
+    else:
+        for k, v in term_freq.items():
+            if isinstance(k, str) is False or isinstance(v, float) is False:
+                return None
+        for k, v in term_freq.items():
+            if isinstance(k, str) is False or isinstance(v, float) is False:
+                return None
+        import math
```

Comment:
all the imports are typically done in the very beginning of the file, not in the middle of the function